css/style.css

```diff
@@ -2,4 +2,52 @@ body {
 	margin-left: 20px;
 	margin-right: 20px;
 	text-align: center;
-}
\ No newline at end of file
+}
+
+header {
+	margin: 3em;
+	width: 100%;
+}
+
+ul {
+
+	display: block;
+	position: relative;
+	list-style: none;
+}
+
+li {
+	display: inline-block;
+	text-align: right;
+}
+
+img {
+	width: 75%;
+	height: 10em;
+}
+
+header img {
+	margin-right: 3em;
+}
+
+main {
+	display: block;
+}
+
+@media only screen and (min-width: 700px) {
+	img {
+		width: 50%;
+		height: 10em;
+	}
+}
```

Comment:
I don't think this media query is actually doing anything.  This rule you are setting is already in place for anything over 400px.